appveyor.yml

```diff
@@ -50,7 +50,7 @@ build_script:
   - "python -c \"import awkward; print(awkward.__version__)\""
   - "pip install \"uproot-methods>=0.7.0\""
   - "python -c \"import uproot_methods; print(uproot_methods.__version__)\""
-  - "pip install cachetools lz4 mock xxhash"
+  - "pip install cachetools zstd mock xxhash"
```

Comment:
zstd shouldn't replace lz4; both should be added here.

Reply:
@jpivarski sorry, it was definitely a typo!